### PR TITLE
B:18520 Update Nav Links To Return To Move Page(MERGE AFTER B-18654 & B-18913)(READY)

### PR DESCRIPTION
**Pull request by TevinAdams**
## [Integration MR](https://github.com/transcom/mymove/pull/12197)

MERGE AFTER 18654 & 18913 READY
## Summary

Went through the milmove multimove workflows and updated the page navigations to route to customerRoutes.MOVE_HOME_PAGE (The Move) where needed. Let me know if I missed any workflows or navigations.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmove app as a service member
2. Go through the workflow of creating a user then a move. Go through the workflow of creating shipments.
3. Go through all workflows to catch where anywhere in the workflows navigations it should route back to the move.
4. Go through the edit workflows and backing out of the edit workflows.
5. Go through the workflows saving changes.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: config/env/stg.app.env [hunk 1]
```diff
@@ -33,5 +33,5 @@ TELEMETRY_ENDPOINT=localhost:4317
 TELEMETRY_USE_XRAY_ID=true
 TLS_ENABLED=true
 SERVE_PRIME_SIMULATOR=true
-FEATURE_FLAG_MULTI_MOVE=false
-FEATURE_FLAG_COUNSELOR_MOVE_CREATE =false
\ No newline at end of file
+FEATURE_FLAG_MULTI_MOVE=true
```
Comment (traskowskycaci) [inline config/env/stg.app.env:36]: Should this be true or false?

Comment (traskowskycaci) [reply config/env/stg.app.env:36]: Ready to give the green light other than this, not sure if it's supposed to be on

Comment (danieljordan-caci) [reply config/env/stg.app.env:36]: I'm with ya here. @deandreJones wanna chime in here? Are we rockin' forward with multi moves being turned on in staging/main?

Comment (cameroncaci) [reply config/env/stg.app.env:36]: Keep in mind, the environment variables are only utilized as a fallback for when Flipt is down or when running server tests in the pipeline

Comment (cameroncaci) [reply config/env/stg.app.env:36]: The CircleCI pipeline cannot see the Flipt server

Comment (deandreJones) [reply config/env/stg.app.env:36]: off

Comment (cameroncaci) [reply config/env/stg.app.env:36]: He has spoken. @TevinAdams gotta update the integration branch one too

Comment (TevinAdams) [reply config/env/stg.app.env]: @deandreJones @cameroncaci @danieljordan-caci @traskowskycaci  Done. Will be making an integration PR with this same change.

Comment (traskowskycaci) [reply config/env/stg.app.env]: Sorry this one slipped off my radar Tevin. I see some of the FEATURE_FLAG_MULTI_MOVE are set to true still. Do we want any trues still in here as we go to main? @TevinAdams

Comment (TevinAdams) [reply config/env/stg.app.env]: @traskowskycaci the feature flag changes that are left is for Circle CI. This allows the playwright tests for Multi Move to run.

Comment (traskowskycaci) [reply config/env/stg.app.env]: Ok, sounds good!

**Review by traskowskycaci — APPROVED** [review]

**Review by danieljordan-caci — APPROVED** [review]

**Review by cameroncaci — APPROVED** [review]

**Pull request merged**